spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/Spark3SortStrategy.java

```diff
@@ -137,7 +137,9 @@ public RewriteStrategy options(Map<String, String> options) {
       sortedDf.write()
           .format("iceberg")
           .option(SparkWriteOptions.REWRITTEN_FILE_SCAN_TASK_SET_ID, groupID)
-          .option(RewriteDataFiles.TARGET_FILE_SIZE_BYTES, writeMaxFileSize())
+          .option(SparkWriteOptions.TARGET_FILE_SIZE_BYTES, writeMaxFileSize())
```

Comment:
Switched to `SparkWriteOptions` instead of `RewriteDataFiles` here. I think it was a typo.